docs/cookbook/fromLegacyDate.mjs

```diff
@@ -0,0 +1,31 @@
+const legacyDate = new Date('1970-01-01T00:00:01Z');
+const instant = legacyDate.toTemporalInstant();
+
+assert.equal(instant.toString(), '1970-01-01T00:00:01Z');
+
+// If you need a ZonedDateTime, use the toZonedDateTime() or
+// toZonedDateTimeISO() method of the resulting Instant.
+
+// You will need to specify a time zone. You might not have time
```

Comment:
Per discussion lower-down below, I don't think we should be pointing new-to-Temporal users to creating ZDT instances in the UTC time zone. That's an advanced use case. If they don't know the TZ and want to work in UTC, then they should really be just using `Temporal.Instant`, not ZDT.

```suggestion
// You will need to specify a time zone, because legacy Date does not
// store the time zone used when it was created.
```

Reply:
I went back and forth on this. The cookbook should definitely not exclude advanced use cases, we intentionally have several other advanced things in there. In the end I thought maybe it's better to just mention in a comment 'you might have to use UTC, but consider using Instant instead'. I wouldn't want to leave it out altogether because otherwise we talk about this distinction between legacy Date's non-UTC and UTC methods but leave people hanging as to what they need to do when it comes to the latter.

I do agree with you that we should just skip converting between legacy Date and Plain types altogether in this example. So I'll even leave out the PlainDateTime examples instead of changing them to PlainDate ones. No need to have that code available to get copy-pasted into places where it's not supposed to be.